README.TXT

```diff
@@ -1,237 +1,246 @@
+
+[Full documentation](https://businessdirectoryplugin.com/knowledge-base/?utm_source=wprepo&utm_medium=link&utm_campaign=liteversion) is also available.
+
+== Screenshots ==
+
+[See feature lists and screenshots](https://businessdirectoryplugin.com/features/?utm_source=wprepo&utm_medium=link&utm_campaign=liteversion)
+
+== Frequently Asked Questions ==
+
+= What types of directories can I build? =
+Our directory website builder software combined with our add-on modules is the most powerful directory plugin on the market. The options are limitless. Here are a few types of directories you can create:
+
+* Business listings
+* Company directory
+* Team directory
+* Member directory
+* Staff directory / Employee directory
+* Hotel directory
+* Medical directory
+* Restaurant directory
+* Church directory
+* Travel review directories
+* Software review sites
+* Professional directory
+* Local provider listings (like a Yelp clone)
+* A Yellow Pages directory
+* Online magazine listings
+* Website listings
+* Book review sites
+
+= Can I accept paid and free listings? =
+Yes it can. You can do this by creating plans for 0.00 and some other paid amount at the same time.
+
+= Do you have video tutorials for building WordPress directories? =
+Yes we do. They can be found [here](https://businessdirectoryplugin.com/video-tutorials/?utm_source=wprepo&utm_medium=link&utm_campaign=liteversion).
+
+= Will this plugin work with my theme? =
+Our WP directory plugin will work with any well-written, WP compliant theme. Generally speaking, paid themes tend to be higher quality than free ones.
+
+Whatever theme you pick, we recommend using the Theme Check plugin (available for free on WordPress.org) to validate a theme. The more errors and warnings you see on the theme when you run that check, the worse the theme is. You can read more about it at https://wordpress.org/plugins/theme-check/
+
+= Can I translate my web directories? =
+Yes! We offer many user-provided translations under the /languages folder. It is already available in the following languages:
+
+* Spanish
+* French
+* German
+* Russian
+* Italian
+* Polish
+* Arabic
+* Swedish
+
+And you can also easily [translate it yourself](https://translate.wordpress.org/projects/wp-plugins/business-directory-plugin/).
+
+= Is this plugin compatible with WordPress Multisite (WPMU)? =
+Yes it is. However, you cannot "network activate" the plugin (as this will share the database). Activate it on only the sub sites you need it.
+
+This can be done under Plugins -> Add New as the Administrator user. Do not "network activate" as the "super admin".
+
+== Changelog ==
+= 6.3.2 =
+* Update: Editor role can manage Business Directory Plugin.
+* Update: Editor, Author, and Contributor roles can see a better design now.
```

Comment:
Here are a few changes to help make the changes more clear. For fixes, I like to use past tense so it's clear the issue no longer exists.

* Update: Users with the Editor role can manage listings in the admin area.
* New: The default theme styles are now minified.
* Fix: Editors, Authors, and Contributors see the navigation sidebar now
* Fix: Editors were getting an extra link in the admin menu that led to a 404 page.
* Fix: On some sites, admin pages were timing out when loading the menu icons.
* Dev/update: All PHP codes are beautified

Can you tell me what this one means?
* Fix: Displaying listing titles when a theme disables the single-page title fixed.

Reply:
The related issue is:
https://github.com/Strategy11/business-directory-premium/issues/44

Reply:
So maybe this?
Fix: Listing titles were not always showing in the Enfold theme